src/app/app.component.ts

```diff
@@ -12,14 +12,103 @@ import { Component, OnInit } from '@angular/core';
 })
 export class AppComponent implements OnInit {
   readonly DEFAULT_BANNER_DESCRIPTION: string = "Safer, faster, and more stable"
+  readonly ANDROID_CHROME_CLICK_DESCRIPTION = "Click to open link with Banana browser"
+
+  readonly PACKAGE_ID = "org.triple.banana"
+  readonly DEFAULT_WAIT_TIME_MS = 300
+
+  readonly TRIPLE_BANANA_DEV_URL = "https://triplebanana.dev"
+  readonly PLAYSTORE_URL = "https://play.google.com/store/apps/details"
+
   bannerDescription = this.DEFAULT_BANNER_DESCRIPTION
 
+  private targetUrl: URL = new URL(this.TRIPLE_BANANA_DEV_URL)
+  private targetUrlOnly: string = ""
```

Comment:
Just query: In Angular, skipping semicolon is general? (If so, please keep a consistency)